src/oop/object_interactions/medium/q3/QuestionText.md

```diff
@@ -0,0 +1,53 @@
+# oop-object_interactions-medium-q3
+
+- Java question
+
+## Question Text
+
+You work at a printing company and your summer job is to make copies of original works. Make a **Newspaper** class that
+has attributes to store the year, month, and day of the print as well as the story.
+
+## Newspaper Class
+
+The `Newspaper` class currently has the following items:
+
+- Attributes:
+    - An `int` called **myYear**
+    - An `int` called **myMonth**
+    - An `int` called **myDay**
+    - A `String` called **myStory**
+    - None of these attributes should be accessible from other classes.
+
+- Constructors:
+    - A constructor with no parameters that initializes all the int attributes to -1 and the **myStory** to an
+      empty `String`.
+    - It is accessible from other classes.
+
+- Methods:
+    - Getter methods for **myStory**, **myYear**, **myMonth**, and **myDay** attributes that are accessible from other
+      classes.
+
+You must add the following items to the class:
+
+- Constructors:
+    - A constructor that takes three `int` and one `String` in that order and initialized the **myYear**, **myMonth**,
+      **myDay**, and **myStory** attribute in that order.
+    - It should be accessible from other classes.
+
+- Methods:
+    - A `makeCopy` method that take another **Newspaper** object and change the attribute of the class to the attributes
```

Comment:
I'm unsure that this should say "class", it doesn't feel like the proper terminology but I may be wrong

Reply:
I think when talking about instances we use the "object" term. Like we don't say that arrays are classes.